app/styles.css

```diff
@@ -0,0 +1,14 @@
+@tailwind components;
+@tailwind utilities;
+
+@media (max-width: 575px) {
+    .max-xs\:e-hidden {
+        display: none;
+    }
+}
+
+@media (max-width: 767px) {
+    .max-sm\:e-hidden {
+        display: none;
+    }
```

Comment:
does this affect anything else?

Reply:
No. We have `max-sm:e-hidden` class only at accordion and I've created `e-` prefix for all the new class declarations to not affect existing styles